src/components/autocomplete/js/autocompleteController.js

```diff
@@ -948,11 +951,37 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
    */
   function updateScroll () {
     if (!elements.li[0]) return;
+    if (mode === MODE_STANDARD) {
+      updateStandardScroll();
+    } else {
+      updateVirtualScroll();
+    }
+  }
+
+  function updateVirtualScroll() {
+    // elements in virtual scroll have consistent heights
     var height = elements.li[0].offsetHeight,
         top = height * ctrl.index,
         bot = top + height,
         hgt = elements.scroller.clientHeight,
```

Comment:
Can you please rename `hgt` to something more explanatory? 
It looks like `height` here should actually be named something like `optionHeight`?
Then `hgt` should actually be `panelHeight`?

Reply:
yup, much better :+1: